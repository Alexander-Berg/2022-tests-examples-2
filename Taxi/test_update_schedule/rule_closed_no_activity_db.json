{
  "descriptors": [
    [
      "moscow",
      "eco",
      "single_ride",
      0,
      100,
      "{activity}",
      false,
      false,
      [
        "sdfsdf"
      ],
      "2022-01-16 21:00:00",
      "2022-01-17 21:00:00",
      "2020-01-20 21:00:00"
    ],
    [
      "moscow",
      "eco",
      "single_ride",
      0,
      50,
      "{activity}",
      false,
      false,
      [],
      "2022-01-16 21:00:00",
      "2023-01-08 21:00:00",
      "2020-01-20 21:00:00"
    ],
    [
      "moscow",
      "eco",
      "single_ride",
      51,
      100,
      "{activity}",
      false,
      false,
      [],
      "2022-01-16 21:00:00",
      "2023-01-08 21:00:00",
      "2020-01-20 21:00:00"
    ]
  ],
  "items": [],
  "tags_masks": [
    [
      "2022-01-02 21:00:00",
      "infinity",
      "single_ride",
      1,
      1,
      [
        "subv_disable_all",
        "subv_disable_single_ride"
      ],
      true
    ]
  ]
}
