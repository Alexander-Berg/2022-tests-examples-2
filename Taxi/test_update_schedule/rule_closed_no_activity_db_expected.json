{
  "descriptors": [
    [
      "moscow",
      "eco",
      "single_ride",
      0,
      49,
      "{activity}",
      false,
      false,
      [],
      "2022-01-12 19:30:00",
      "2022-01-16 21:00:00",
      "2022-01-10 18:03:00"
    ],
    [
      "moscow",
      "eco",
      "single_ride",
      50,
      100,
      "{activity}",
      false,
      false,
      [],
      "2022-01-12 19:30:00",
      "2022-01-16 21:00:00",
      "2022-01-10 18:03:00"
    ],
    [
      "moscow",
      "eco",
      "single_ride",
      0,
      100,
      "{activity}",
      false,
      false,
      [
        "sdfsdf"
      ],
      "2022-01-16 21:00:00",
      "2022-01-17 21:00:00",
      "2020-01-20 21:00:00"
    ],
    [
      "moscow",
      "eco",
      "single_ride",
      0,
      100,
      "{activity}",
      false,
      false,
      [],
      "2022-01-16 21:00:00",
      "2023-01-08 21:00:00",
      "2022-01-10 18:03:00"
    ]
  ],
  "items": [
    [
      7,
      "simple_rule",
      "1",
      253800,
      304800,
      400,
      null,
      null,
      50,
      null,
      false
    ],
    [
      7,
      "simple_rule2",
      "2",
      304800,
      604800,
      400,
      null,
      null,
      null,
      null,
      null
    ],
    [
      9,
      "simple_rule",
      "1",
      253800,
      304800,
      400,
      null,
      null,
      50,
      null,
      true
    ],
    [
      9,
      "simple_rule2",
      "2",
      304800,
      604800,
      400,
      null,
      null,
      null,
      null,
      null
    ]
  ],
  "tags_masks": [
    [
      "2022-01-02 21:00:00",
      "infinity",
      "single_ride",
      1,
      1,
      [
        "subv_disable_all",
        "subv_disable_single_ride"
      ],
      true
    ]
  ]
}
